tests/h/search/core_test.py

```diff
@@ -243,3 +245,17 @@ def test_only_200_replies_are_included(self, pyramid_request, Annotation):
 
         assert len(result.reply_ids) == 3
         assert oldest_reply.id not in result.reply_ids
+
+
+@pytest.fixture(params=['es1', 'es6'])
+def pyramid_request(request, pyramid_request, es_client, es6_client):
+    pyramid_request.es = es_client
+    pyramid_request.es6 = es6_client
+
+    def feature(flag):
+        if flag == 'search_es6':
+            return request.param == 'es6'
+        return True
+
+    pyramid_request.feature = mock.Mock(side_effect=feature)
```

Comment:
I forgot this when I last looked at this PR, but there is a shortcut for setting feature flags in tests:

```py
pyramid_request.feature.flags['flag_name'] = flag_state
```